test/AST-Quake/ranged_for.cpp

```diff
@@ -307,3 +307,48 @@ struct FreeRange {
 // CHECK:             cc.continue %[[VAL_15]] : i64
 // CHECK:           } {invariant}
 // clang-format on
+
+struct FreeRangeChicken {
+  void operator()(cudaq::qvector<> r, int Z, int N, int S) __qpu__ {
+    for (auto i : cudaq::range(Z, N, S)) {
```

Comment:
Is there already a separate issue covering docs updates for this new feature, or should we create a new one? In any case, we should probably specify any limitations w.r.t. `break` if needed.

Reply:
It's in the task list of #1483

Reply:
I'd add a milestone of 0.8, but I cannot.